lib/ansible/modules/cloud/openstack/os_security_group.py

```diff
@@ -126,6 +135,9 @@ def main():
         if state == 'present':
             if not secgroup:
                 secgroup = cloud.create_security_group(name, description)
+                if egress_default_rules == 'absent':
```

Comment:
Maybe I'm missing something, but it looks to me like setting egress_default_rules to absent will remove _all_ rules from the group that was just created (the default set of rules could be different things on different clouds iirc) Perhaps we should just make this a bool called "delete_default_rules" or something - and mention in the doc string that it only has any effect when you are creating a security group.